index.html

```diff
@@ -8,19 +8,55 @@
     <link rel="stylesheet" href="css/index.css">
 </head>
 <body>
-    <img src="images/headshotpic.jpeg" alt="Doina Fitchevici" class="favorite_picture">
+    <header>
+            <nav class="navBar">
+                <div class="menu">
+                    <div id="profile_picture">
+                        <img src="images/headshotpic.jpeg" alt="Doina Fitchevici" class="favorite_picture">
+                    </div>
+                    <div id="profileName">Doina Fitchevici</div>
+                </div>
+            
+                <div class="menu">
+                    <div class="menuItem">
+                        <a href="#about">About</a>
+                    </div>
+                    <div class="menuItem">
+                        <a href="#experience">Experience</a>
+                    </div>
+                    <div class="menuItem">
+                        <a href="#skill">Skills</a>
+                    </div>
+                    <div class="menuItem">
+                        <a href="#connect">GitHub</a>
+                    </div>
+                    <div class="menuItem">
+                        <a href="#leave_a_message">Contact</a>
+                    </div>
+                </div>
+            </nav>
+    </header>
+    
     <main>
-        <section class="section">
+        <pre id="about">About
+            section
+        </pre>
+        <section>
             <h2>About</h2>
             <p>
                 Hello and welcome to my portfolio website! My name is Doina Fitchevici, I am an aspiring front-end web developer, and also a mother of a two-year old boy. Besides enjoying being a mother, I am thrilled to embark on this journey to pursue my passion for web development and share my journey with you.
                 Although I may currently lack professional experience in the field, I have dedicated significant time and effort to learning and honing my skills as a front-end web developer.cd
             </p>
         </section>
-        <section class="section">    
+        <pre id="experience">Experience
+            section
+        </pre>
+        <section>  
             <h2>Experience</h2>
-            <h4>Stay At Home Mom</h4>
-            <h5>Self-Employed</h5>
+            <div class="experience-container"> 
+            <div class="experience-item">
+            <h5>Stay At Home Mom</h5>
+            <h6>Self-Employed</h6>
```

Comment:
```suggestion
            <div class="experience-item">
                <h5>Stay At Home Mom</h5>
                <h6>Self-Employed</h6>
```

Remember to indent. Your IDE might be able to do this automatically.